nextcloud/Readme.md

```diff
@@ -0,0 +1,19 @@
+This directory contains the terraform and heat scripts for running Nextcloud.
+
+To run terraform:
+
+**In the terminal**
+
+```shell
+cd terraform
+terraform init
+terraform plan
+terraform apply --var volume_uuid="<volume id>"
```

Comment:
needs to explain that
1) volume_uuid is optional and allows attaching existing volume (default is to create new volume)
2) ddns parameters, how to provide them and what they do